Formula/breakpad.rb

```diff
@@ -0,0 +1,69 @@
+class Breakpad < Formula
+  desc "Components implement a crash-reporting system."
+  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
+
+  url "https://chromium.googlesource.com/breakpad/breakpad",
+      :using => :git,
+      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
```

Comment:
I'd like to see us ask Google here on their issue tracker and link it in this issue, at least.